### PR TITLE
test: add batch and tda integration tests

### PR DESCRIPTION
**Pull request by cowan-macady**
implement ENG-4048

- add batch ingest tests
- add tda tests
- update proto files

### CODE REVIEW

File: config/config_request.go [hunk 1]
```diff
@@ -279,26 +226,6 @@ func (x *NodeRequest) WithAuditSinkConfig(v *configpb.AuditSinkConfig) *NodeRequ
 	return x
 }
 
-func (x *NodeRequest) WithTokenIntrospectConfig(v *configpb.TokenIntrospectConfig) *NodeRequest {
```
Comment (arxeiss) [inline config/config_request.go:282]: Why it is deleted? I just added that. It is required by Terraform.

File: tda/data_access_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2023 IndyKite
```
Comment (arxeiss) [inline tda/data_access_integration_test.go:1]: 2024

**Review by arxeiss — CHANGES_REQUESTED** [review]

File: config/config_request.go (hunk 1 above)
Comment (cowan-macady) [reply config/config_request.go]: oops! wrong rebase! Fortunately you saw this

File: tda/data_access_integration_test.go (hunk 2 above)
Comment (cowan-macady) [reply tda/data_access_integration_test.go]: updated

**Review by arxeiss — APPROVED** [review]

**Pull request merged**